<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SongInfoDlg</class>
 <widget class="QDialog" name="SongInfoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Song Info</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Form1.qrc">
    <normaloff>:/icons/Form1.png</normaloff>:/icons/Form1.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Song Information:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="8" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Start Offset:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="EditAlbum">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QCheckBox" name="EditEndOffset">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Disc Number:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Album:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QCheckBox" name="EditVolume">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>End Offset:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QComboBox" name="RatingCombo">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QSlider" name="VolumeSlider">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>-15</number>
           </property>
           <property name="maximum">
            <number>15</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="pageStep">
            <number>5</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="AlbumLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="EditTitle">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QCheckBox" name="EditYear">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="ArtistLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QCheckBox" name="EditDisc">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="10" column="2">
          <widget class="QCheckBox" name="EditRating">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Year:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QCheckBox" name="EditStartOffset">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="TitleLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Rating:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="TrackLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="LengthLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Title:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="DiscLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="EditTrack">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Adjust volume:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Artist:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Track:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="YearLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="EditArtist">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Length:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="StartOffsetLineEditMin">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="inputMask">
              <string>000</string>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="StartOffsetLineEditSec">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="inputMask">
              <string>00</string>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="StartOffsetLineEditMS">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="inputMask">
              <string>000</string>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="StartOffsetNow">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Form1.qrc">
               <normaloff>:/icons/icons/jumpto.png</normaloff>:/icons/icons/jumpto.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="9" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="EndOffsetLineEditMin">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="inputMask">
              <string>000</string>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="EndOffsetLineEditSec">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="inputMask">
              <string>00</string>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="EndOffsetLineEditMS">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="inputMask">
              <string>000</string>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="EndOffsetNow">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Form1.qrc">
               <normaloff>:/icons/icons/jumpto.png</normaloff>:/icons/icons/jumpto.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Song Locations:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="LocationsList"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Form1.qrc"/>
 </resources>
 <connections/>
</ui>
